### PR TITLE
Fix regression when original_index = 0

### PR DESCRIPTION
**Pull request by scurest**
Fixes the regression in 葬 (and probably many others :) from #913.

(Sorry!)

### CODE REVIEW

File: src/game_character.cpp [hunk 1]
```diff
@@ -266,6 +266,9 @@ void Game_Character::MoveTypeCustom() {
 					looped_around = true;
 					active_route_index = 0;
 					SetMoveRouteRepeated(true);
+					if (original_index == 0 && looped_around) {
```
Comment (carstene1ns) [inline src/game_character.cpp]: You set `looped_around` to true directly above. Seems like this check is not needed then.

Comment (scurest) [inline src/game_character.cpp]: Oops.

**Comment by Ghabry** [comment]
Jenkins: Test this please

**Comment by Ghabry** [comment]
Will update the compiler to "Update 3", maybe that helps o.O

**Comment by Ghabry** [comment]
Jenkins: Test this please

**Pull request merged**